src/components/SQForm/SQFormCheckboxGroupItem.tsx

```diff
@@ -14,6 +13,23 @@ const useStyles = makeStyles(theme => ({
   }
 }));
 
+interface SQFormCheckboxGroupItemProps {
+  /** The name of the group this checkbox is a part of */
+  groupName: string;
+  /** Label for the checkbox */
+  label: string;
+  /** Value for the checkbox */
+  value: string | boolean | number;
+  /** Function to call when input value is changed */
+  onChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
```

Comment:
We can use `CheckboxProps` here as well, to get the `onChange` type that `<Checkbox />` expects: 
```ts
onChange: CheckboxProps['onChange'];
````

Reply:
@codyarose The useForm component does not support that. I have tried and it gives me this error 
![image](https://user-images.githubusercontent.com/81575187/125783718-72ef9530-8828-40fd-b782-4f4c868e44cb.png)

Reply:
Ah, right. My mistake, that should be possible after I add the useForm generics.